esp/templates/survey/survey_form.html

```diff
@@ -1,125 +1,56 @@
+    Please select another survey component below or come back later.
+    </span>
+    <br /><br />
+    {% endif %}
+    This survey consists of a survey component for each class you are registered for and a general program survey component. You may fill out a class survey once the class has begun and you may fill out the general program survey once your first class has begun. If no survey components are available, please come back later.
+    </p>
+    <table class="fullwidth">
+        <tr>
+            <th>Survey Component</th>
+            <th>Status</th>
+        </tr>
         {% for sec in sections %}
-          <table class="fullwidth">
-           <tr>
-                <th>Questions for {{ sec.emailcode }}: {{ sec.title }} ({% for tm in sec.friendly_times %}{% if forloop.counter > 1 %},{% endif %}{{ tm }}{% endfor %})</th>
-            </tr>
-            </table>
-            {% for question in perclass_questions %}
-                {{ question.render|substitute:sec }}
-            {% endfor %}
+        <tr>
+            <td class="underline" align="center">{{ sec }}</td>
+            <td class="underline" align="center">{% if not sec.started %}<span style="color: grey">Not available</span>{% elif sec.completed %}<span style="color: green">Completed</span>{% else %}<a class="btn fancybutton no-dec" href="{{ request.path }}?sec={{ sec.id }}">Take Survey</a>{% endif %}</td>
+        </tr>
         {% endfor %}
-    {% endif %}
+        <tr>
+            <td class="underline" align="center">General Program Survey</td>
+            <td class="underline" align="center">{% if not general_available %}<span style="color: grey">Not available</span>{% elif general_done %}<span style="color: green">Completed</span>{% else %}<a class="btn fancybutton no-dec" href="{{ request.path }}?general">Take Survey</a>{% endif %}</td>
+        </tr>
+    </table>
+{% else %}
+    <form method="post" action="{{ request.get_full_path }}">
+    {% csrf_token %}
+    {% if perclass_questions %}
+        {% for question in perclass_questions %}{{ question.render|substitute:section }}{% endfor %}
+    {% elif questions %}
+        {% for question in questions %}{{ question.render|uselist:classes }}{% endfor %}
     {% endif %}
 
-{% endif %}
-
-<table class="fullwidth">
-    <tr>
-        <th>Submit</th>
-    </tr>
-    <tr>
-        <td align="center">
-            Click the button below to finish your survey.  Thank you so much for your time!<br /><br />
-            <input type="Submit" class="fancybutton" value="Submit Survey" /><br />
-        </td>
-    </tr>
-</table>
+    <table class="fullwidth">
+        <tr>
+            <th>Submit</th>
+        </tr>
+        <tr>
+            <td align="center">
+                Click the button below to finish your survey. Thank you so much for your time!<br /><br />
+                <input type="Submit" class="fancybutton" value="Submit Survey" /><br />
```

Comment:
On the webapp, the "Submit Survey" button is a different font than everything else. If it's an easy fix, could we make it match? Otherwise it's fine.